ui-v2/app/components/role-selector.js

```diff
@@ -1,25 +1,25 @@
 import ChildSelectorComponent from './child-selector';
 import { inject as service } from '@ember/service';
 import { get, set } from '@ember/object';
-
 import { alias } from '@ember/object/computed';
 
 import { CallableEventSource as EventSource } from 'consul-ui/utils/dom/event-source';
 
 export default ChildSelectorComponent.extend({
   repo: service('repository/role/component'),
+  dom: service('dom'),
   name: 'role',
   type: 'role',
   classNames: ['role-selector'],
   state: 'role',
+  // You have to alias data
+  // is you just set it it loses its reference?
```

Comment:
This sentence seems like parts are missing.

Reply:
`is = if`, 😂 my bad will update later